docs/contributing-analyzer.md

```diff
@@ -12,6 +12,8 @@ In general, it is best to run commands from the Visual Studio Developer Command
 
 ## Developing with Visual Studio 2022 or Rider
 
+Visual Studio 2022 version 17.5+ is required to build the project (due to source generators) and run the unit tests.
```

Comment:
FIXME: This will most likely need to be 17.6+ once released.
And likely, we'll need to bump the Roslyn references to 4.6.0 too.

Reply:
17.5.4 works, 17.6 will look nicer once released.